arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -124,6 +124,24 @@ Or:
   chmod -fR 777 /data/arbitrum
   ```
 
+### Note on Sequencer feed
+
+The Arbitrum node requires a feed to receive real time ordered transactions from the sequencer. If you don't set feed input url, your node will listen to parent chain inbox contract to get ordered transaction which will cause your node to be unable to synchronize the latest state.
+
+To make your node listen to sequencer feed, you need to set those configurations to your sequencer node:
```

Comment:
I would maybe change this a bit, so it's clear that this is something that needs to be configured in the sequencer, not the orbit full node. Otherwise, since this document is mainly targetted at node runners (not chain owners), they might be confused, and think that they need to add these to their nodes.

"In order for a node to read the sequencer feed, the chain's sequencer needs to be configured with the following parameters:"